btbridged.c

```diff
@@ -273,7 +271,7 @@ static int method_send_message(sd_bus_message *msg, void *userdata, sd_bus_error
 		r = 0;
 		goto out;
 	}
-	r = sd_bus_message_read(msg, "yyyyy", &netfn, &lun, &seq, &cmd, &cc);
+	r = sd_bus_message_read(msg, "yyyyy", &seq, &netfn, &lun, &cmd, &cc);
```

Comment:
Does this make sense to be a message_read?  Should it be a message_append?  This should be yyyyay.  The code will send data packet to the host.